src/datautils.py

```diff
@@ -186,9 +206,8 @@ def get_loaders(name, nsamples=128, seed=0, seqlen=2048, eval_mode=False, model_
     set_seed(seed)
 
     # for pre-tokenized datasets
-    if name.lower() == "pajama":
-        data = torch.load(f"./data/red_pajama_n=1024_{seqlen}_context_length.pth")[:nsamples]
-    elif name.lower() == "refinedweb":
+
+    if name.lower() == "refinedweb":
         data = torch.load("./data/refined_web_n=128.pth")[:nsamples]
```

Comment:
I'd recommend that we remove refinedweb since we don't even know what it was tokenized with

Reply:
will do

Reply:
Done

Reply:
Do we really intend to work with Falcon models?

Reply:
we may eventually need 'em, but the user can still run falcon via DATASET=./data/... as long as they know what they're doing.

My concern is that someone might accidentally try training llama/mistral on main.py "refinedweb" without realizing it's model-specific